docs/diagrams/FileManagerDiagram.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
+!include Styles.puml
+'https://plantuml.com/class-diagram
+
+Package "filemanager"  <<Rectangle>>{
+     Class Storage COLOR_STORAGE
+     Class InstrumentDecoder COLOR_STORAGE
+     Class InstrumentEncoder COLOR_STORAGE
+     Class CryptoDecoder COLOR_STORAGE
+     Class StockDecoder COLOR_STORAGE
+     Class EtfDecoder COLOR_STORAGE
+     Class ForexDecoder COLOR_STORAGE
```

Comment:
Can we have like XYZDecoder instead of all classes. Reason is that you also need to draw dependencies between the classes here and instrument and instrumentManager class. So this helps to reduce cluttering